### ISSUE TITLE
Better polls (STV, etc.)

### ISSUE THREAD

**Issue by ericthelemur** [UWCS/apollo#126]

**Comment by raven0034** [comment]
Better done via Vote rather than duplicating functionality into Apollo.

**Comment by ericthelemur** [comment]
True, most of this issue was really waiting for me to port the voting code from old d.py, but not used enough to bother (especially now actual discord polls exist)
